MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
```

Comment:
Similar to the `roles` we had earlier...is this something that should go in now, or should we wait until we need it?

I'm kinda on the fence, could see arguments for both.

If we supported it, would it be required?